spec/spec_helper_acceptance_local.rb

```diff
@@ -191,3 +191,30 @@ def mod_supported_on_platform?(mod)
   return false if ENV['DISABLE_MOD_TEST_EXCLUSION']
   ApacheModTestFilterHelper.instance.mod_supported_on_platform?(mod)
 end
+
+RSpec::Matchers.define :apply do
```

Comment:
IMHO this matcher should have been implemented in beaker-rspec and then in puppet_litmus. It provide a much more native RSpec experience in writing tests.